activate-virtualenv.sh

```diff
@@ -25,11 +25,17 @@ if ! type "workon" &>/dev/null; then
   source $virtualenvwrapper_path
 fi
 
+if ! which python2.7 &>/dev/null; then
```

Comment:
The assumption is that if there is a `python2.7` in our `PATH` somewhere, it is expected to actually point to a Python 2.7 interpreter. I think that's a reasonable assumption, unless I'm misunderstanding the concern.

Otherwise, I think if you've got a slightly esoteric Python setup that you're afraid this will break, it's easy enough to make the virtualenv yourself. These scripts are only really intended for local development anyway. I know for a fact the only one we use as part of the deployment process is `frontend.sh`. 

That all being said, I think we all agree that (1) these scripts are fragile, and make too many assumptions as is, (2) are hard to troubleshoot when they do fail, and (3) the real problem here is our dependency on a Python 3-incompatible library.